src/js_native_api_types.h

```diff
@@ -115,4 +115,22 @@ typedef struct {
   napi_status error_code;
 } napi_extended_error_info;
 
+typedef enum {
+  napi_key_all,
+  napi_key_own
+} napi_key_collection_mode;
+
+typedef enum {
+  napi_key_writable = 1,
```

Comment:
Is a `napi_key_all_properties = 0` required to represent a default all-inclusive filter mode?